### PR TITLE
more fixes based on collaborative review

### PR DESCRIPTION
**Pull request by TomasTomecek**
* introduce a func to detect if selinux is disabled
* implement kill for docker containers
* docs updates

### CODE REVIEW

**Comment by psklenar** [comment]
## jenkins/superCI comment
* Status: success, return code = __0__ 
* Result url: __[https://da.gd/T5M7](https://da.gd/T5M7)__
* Comment: __See details for logs__

File: conu/apidefs/container.py [hunk 1]
```diff
@@ -239,9 +239,10 @@ def stop(self):
 
     def kill(self, signal=None):
         """
-        kill this container
+        kill this container (bear in mind that the process won't have time to shutdown properly
```
Comment (dhodovsk) [inline conu/apidefs/container.py]: Just wondering. Killing isn't the only ability of this method. docker kill --help says `Kill one or more running containers` but I think I like `Send a signal to..` better. @TomasTomecek what do you think?

Comment (TomasTomecek) [reply conu/apidefs/container.py]: I agree with you, will update.

**Comment by TomasTomecek** [comment]
rebased, updated

**Comment by psklenar** [comment]
## jenkins/superCI comment
* Status: failure, return code = __1__ 
* Result url: __[https://da.gd/seMp](https://da.gd/seMp)__
* Comment: __See details for logs__

**Comment by psklenar** [comment]
## jenkins/superCI comment
* Status: success, return code = __0__ 
* Result url: __[https://da.gd/rxrm](https://da.gd/rxrm)__
* Comment: __See details for logs__

**Pull request merged**